tests/Parser/SalesPricesTest.php

```diff
@@ -41,27 +56,59 @@ public function testParse($data, $expectedResult)
         $this->assertSame($expectedResult, $salesPricesMock->getResults());
     }
 
-    public function getRRPProvider()
+    public function providerGetDefaultPrice()
+    {
+        return array(
+            array(
+                array(),
+                false
+            ),
+            array(
+                array('default' => array(2, 4)),
+                2
+            )
+        );
+    }
+
+    /**
+     * @dataProvider providerGetDefaultPrice
+     */
```

Comment:
- [ ] Add the argument definitions to this doc block.

- [ ] Add the type hint for `$parsedData` e.g. `array $parsedData`.